src/app/components/login/login.component.html

```diff
@@ -1,113 +1,105 @@
+            <a href="https://github.com/Mijago/D2ArmorPicker" target="_blank"
+                >in the Github Repository</a
+            >!
         </p>
-      </mat-expansion-panel>
-
-      <mat-expansion-panel class="mat-elevation-z11">
-        <mat-expansion-panel-header>
-          <mat-panel-title>
-            Result Display
-          </mat-panel-title>
-          <mat-panel-description>
-            What you will get from this tool.
-          </mat-panel-description>
-        </mat-expansion-panel-header>
         <p>
-          D2ArmorPicker will show you a table with many results.
-          You can sort and filter it, and then you click on one row to see which items are needed for your desired stats.
-          <img class="image-example" src="assets/resultOverviewWithFlex.png">
-          <br>
-          <br>
-          The page will also give you a detailed description of what you have to do in order to get to your desired build.
-          The following screenshot shows one of the more detailed descriptions:
-          <img class="image-example" src="assets/info/whattodonow.png">
+            D2ArmorPicker uses the Bungie.net api and it's OAuth authorization. You log in on an
+            official Bungie.net page and D2ArmorPicker receives a temporary login token from Bungie.
+            That means it does not receive your credentials.
         </p>
-      </mat-expansion-panel>
+    </mat-card-content>
+    <mat-card-actions>
+        <button (click)="startLogin()" color="primary" mat-raised-button>Login</button>
+    </mat-card-actions>
+</mat-card>
+
+<mat-card>
+    <mat-card-title> Examples </mat-card-title>
+    <mat-card-subtitle>Some examples to show what D2ArmorPicker can do.</mat-card-subtitle>
+    <mat-card-content>
+        <mat-accordion>
+            <mat-expansion-panel class="mat-elevation-z11">
+                <mat-expansion-panel-header>
+                    <mat-panel-title> Stat Selection </mat-panel-title>
+                    <mat-panel-description> The heart of this tool. </mat-panel-description>
+                </mat-expansion-panel-header>
+                <p>
+                    First, you are able to select the stats you want. The tool will automatically
+                    parse all your armor (including your vault and your postmaster), and then show
+                    you which stat selections are even possible with your current gear. The tool
+                    also shows you all possible loadouts with three or four stats at 100!
+                    <img class="image-example" src="assets/info/statselection.png" />
+                    <br />
+                    <br />
+                    There are also additional settings. You can add stasis fragments and negative
+                    and positive stat mods to your builds. Also, you can select 0-waste builds (so
+                    only flat numbers). D2ArmorPicker will also add stat mods to reduce wasted stats
```

Comment:
0-waste is disabled right now

Reply:
Also best to make a new ticket for the login page text, as changing it is not the scope of this PR.